pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -261,14 +258,14 @@ func (exp *flowExporter) sendFlowRecords() error {
 		// (activeFlowTimeout or idleFlowTimeout) are met. A flow is considered
 		// to be idle if its packet counts haven't changed since the last export.
 		if time.Since(record.LastExportTime) >= exp.idleFlowTimeout {
-			if ((record.Conn.OriginalPackets <= record.PrevPackets) && (record.Conn.ReversePackets <= record.PrevReversePackets)) || !record.Conn.IsPresent {
+			if ((record.Conn.OriginalPackets <= record.PrevPackets) && (record.Conn.ReversePackets <= record.PrevReversePackets)) || flowexporter.IsConnectionDying(&record.Conn) {
 				// Idle flow timeout
 				record.IsActive = false
 				recordNeedsSending = true
 			}
 		}
-		if time.Since(record.LastExportTime) >= exp.activeFlowTimeout {
-			// Active flow timeout
+		if time.Since(record.LastExportTime) >= exp.activeFlowTimeout || flowexporter.IsConnectionDying(&record.Conn) {
```

Comment:
Why is this needed? As idleFlowTimeout happens earlier than activeFlowTimeout, flow record for the connection in "DYING" state would have already been sent.

Reply:
Forgot to remove this change. Updated. Thanks!